skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -269,6 +269,33 @@ def test_binmap3d():
     rbinmap2 = binstat2.binmap
 
     assert_array_almost_equal(rbinmap1[0][::1000], rbinmap2[0][::1000])
-    assert_array_almost_equal(rbinmap1[0][::1000], np.array([1, 10,  9,  8, 7,
-                                                             6,  5,  4,  3, 2,
+    assert_array_almost_equal(rbinmap1[0][::1000], np.array([1, 10, 9, 8, 7,
+                                                             6, 5, 4, 3, 2,
                                                              1]))
+
+
+@pytest.mark.parametrize(['stat', 'stat_f'], stats_list)
+def test_BinnedStatistics1D_mask(stat, stat_f):
```

Comment:
Seems like another simple test you could do to make sure the mask is working would be to:

1. make an array of ones
2. make a mask that masks half of the array
3. run through the binning code
4. make sure that the resultant binned values are equal to half the size of the array.

That might be easier to reason about instead of somewhat duplicating the ``test_BinnedStatistics1D`` function

Reply:
I the current test is fine, but perhaps documentation should be added to make it clear that the point of this test is to compare scipy to `binned_statistic`?

Reply:
@jrmlhermitte check out [this other test in the test suite](https://github.com/scikit-beam/scikit-beam/blob/master/skbeam/core/accumulators/tests/test_binned_statistic.py#L200-L222). It is quite similar to this one that seems like it is doing the heavy lifting of comparing scipy to binned_statistic. Given that test already exists it seems like we just want to make sure the masking code is working properly which could be a much simpler test with no duplicated code.  Another possible avenue for testing the masking code would be to update the linked test to take a mask parameter as part of the pytest parametrize fixture and then there'd be no duplicated code in two different tests